website/templates/project/addon/widget.mako

```diff
@@ -9,14 +9,17 @@
 
                 </div>
             </div>
+    % else:
+        <div>
+    % endif
 
     % if complete:
 
         <div class="panel-body">
             ${self.body()}
         </div>
 
-    % else:
+    % elif not complete and 'admin' in user['permissions']:
```

Comment:
No, I'm asking under what circumstances that block should show so we can figure out what the logic should be. Could be the `or` is wrong, could be the `write` is wrong, could be both, could be something else.